types/koa/index.d.ts

```diff
@@ -453,32 +453,32 @@ declare class Application extends EventEmitter {
         port?: number,
         hostname?: string,
         backlog?: number,
-        listeningListener?: () => void,
+        listeningListener?: Function,
     ): Server;
     listen(
         port: number,
         hostname?: string,
-        listeningListener?: () => void,
+        listeningListener?: Function,
     ): Server;
     listen(
         port: number,
         backlog?: number,
-        listeningListener?: () => void,
+        listeningListener?: Function,
     ): Server;
-    listen(port: number, listeningListener?: () => void): Server;
+    listen(port: number, listeningListener?: Function): Server;
     listen(
         path: string,
         backlog?: number,
-        listeningListener?: () => void,
+        listeningListener?: Function,
     ): Server;
-    listen(path: string, listeningListener?: () => void): Server;
-    listen(options: ListenOptions, listeningListener?: () => void): Server;
+    listen(path: string, listeningListener?: Function): Server;
+    listen(options: ListenOptions, listeningListener?: Function): Server;
     listen(
         handle: any,
         backlog?: number,
-        listeningListener?: () => void,
+        listeningListener?: Function,
```

Comment:
Please don't use `Function` as it doesn't specify the types of its arguments, so any function is assignable to it. If there's something this callback provides that isn't there currently you could add it as a new parameter.

Reply:
`() => void` _is_ the correct type for any function that takes no arguments and whose return value is ignored 🤔